src/JsonSchema/Fuzz.elm

```diff
@@ -76,12 +78,12 @@ objectFuzzer objectSchema =
             Maybe.Extra.isJust objectSchema.minProperties
                 || Maybe.Extra.isJust objectSchema.maxProperties
     in
-    if unfuzzable then
-        Fuzz.invalid "Fuzzing minProperties or maxProperties is currently not supported."
-    else
-        List.map propertyFuzzer objectSchema.properties
-            |> Fuzz.Extra.sequence
-            |> Fuzz.map (Maybe.Extra.values >> Encode.object)
+        if unfuzzable then
```

Comment:
It looks like you're using a different version of elm-format than the rest of this project. Could you run version `0.7.0-exp` working on this repo? That will result in the same indentation behaviour as the rest of the code and smaller diffs :).